src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -542,33 +542,17 @@ private static bool CheckParameterRefEscape(SyntaxNode node, BoundParameter para
 
             // byval parameters can escape to method's top level.
             // others can be escape further, unless they are ref-like.
-            if (escapeTo == Binder.ExternalScope)
+            if (escapeTo == Binder.ExternalScope && parameterSymbol.RefKind == RefKind.None)
             {
-                if (parameterSymbol.RefKind == RefKind.None)
+                if (checkingReceiver)
                 {
-                    if (checkingReceiver)
-                    {
-                        Error(diagnostics, ErrorCode.ERR_RefReturnParameter2, parameter.Syntax, parameterSymbol.Name);
-                    }
-                    else
-                    {
-                        Error(diagnostics, ErrorCode.ERR_RefReturnParameter, node, parameterSymbol.Name);
-                    }
-                    return false;
+                    Error(diagnostics, ErrorCode.ERR_RefReturnParameter2, parameter.Syntax, parameterSymbol.Name);
                 }
-
-                if (parameterSymbol.Type?.IsByRefLikeType == true)
```

Comment:
This remained by accident from the previous safety rules. 
Byref parameters are now ref-returnable regardless of their type.